notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -32,21 +32,22 @@
         "<table align=\"left\">\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    delete_bucket = False
</pre><p>Actually, please revert to the previous form as we need to delete buckets in testing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
@ivanmkc 

Not sure what you mean, we already have the delete bucket if testing:

if delete_bucket or os.getenv("IS_TESTING"):
    ! gsutil rm -r $BUCKET_URI